### ISSUE TITLE
Provide link to skip repetitive nav (including tabzilla)

### ISSUE THREAD

**Issue by stenington** [mozilla/openbadges-backpack#522]
A hidden skip link should be made available to screen readers, and/or use the `<main>` element or ARIA attribute `role=main`.
